python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -171,7 +175,7 @@ def __init__(
 
         self.n_estimators = n_estimators
         self.n_estimators_per_worker = list()
-
+        self._multi_class = False
```

Comment:
For discussion: Should we save n_classes instead? Seems like richer info and still covers the same thing. Could always have a wrapper method or property that converts it to boolean